src/aliases.rs

```diff
@@ -49,5 +49,17 @@ pub type U2048 = Uint<2048, 32>;
 /// [`Uint`] for `4096` bits.
 pub type U4096 = Uint<4096, 64>;
 
+/// [`Bits`] for `128` bits.
+pub type H128 = Bits<128, 2>;
+
+/// [`Bits`] for `160` bits.
+pub type H160 = Bits<160, 3>;
+
+/// [`Bits`] for `256` bits.
+pub type H256 = Bits<256, 4>;
+
+/// [`Bits`] for `512` bits.
+pub type H512 = Bits<512, 8>;
```

Comment:
I don't like the `H` prefix because these are bit arrays, they have little to do with hashes.